packages/twenty-front/src/modules/workflow/hooks/useStepsOutputSchema.ts

```diff
@@ -0,0 +1,35 @@
+import { useAvailableComponentInstanceIdOrThrow } from '@/ui/utilities/state/component-state/hooks/useAvailableComponentInstanceIdOrThrow';
+import { WorkflowVersionComponentInstanceContext } from '@/workflow/states/context/WorkflowVersionComponentInstanceContext';
+import { stepOutputSchemaFamilySelector } from '@/workflow/states/selectors/stepOutputSchemaFamilySelector';
+import { useRecoilCallback } from 'recoil';
+import { isDefined } from 'twenty-shared';
+
+export const useStepsOutputSchema = (instanceIdFromProps?: string) => {
```

Comment:
The hook seems never to be called with the prop `instanceIdFromProps`. Is the prop required? I don't understand what `instanceIdFromProps` means.